manifests/profile/master/install.pp

```diff
@@ -12,6 +12,12 @@
     $puppetserver_service = 'puppetserver'
   }
 
+  package { 'toml':
```

Comment:
If possible I think we should only install this if `puppet_metrics_dashboard::enable_ldap_auth` is true